var/spack/repos/builtin/packages/dbow2/package.py

```diff
@@ -19,3 +19,4 @@ class Dbow2(CMakePackage):
     depends_on('opencv+calib3d+core+features2d+highgui+imgproc')
     depends_on('boost')
     depends_on('dlib')
+    depends_on('eigen', type='build')
```

Comment:
Does this dependency apply to all available versions of the software?

Reply:
Yes, it is required for all versions.

Reply:
Additional Information:
dbow2 cannot be installed until pull request #21718 is merged.

Reply:
#21718 has already been merged.